easybuild/easyconfigs/b/Boost/Boost-1.55.0-goolf-1.7.20.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Boost'
+version = '1.55.0'
+
+homepage = 'http://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%%(namelower)s_%s.tar.gz' % '_'.join(version.split('.'))]
+
+configopts = '--without-libraries=python'
+
+dependencies = [('bzip2', '1.0.6')]
+
+# also build boost_mpi
+boost_mpi = True
+
+osdependencies = [('zlib-devel','zlib1g-dev')]
```

Comment:
right below `dependencies` line

in fact, we should look into changing this to a proper `zlib` dependency...

the `TopHat` easyconfig included here suggest this should be `zlib/1.2.8`, but this may cause conflicts in other easyconifgs using `Boost` as a dep...

@pescobar: up for giving setting `zlib/1.2.8` as a dep here a try?